ftw/poodle/browser/poodletable.py

```diff
@@ -43,6 +44,15 @@ def isCurrentUser(self, userid):
         # also add is poodle active check
         return user.id == userid and self.is_active()
 
+    def isOwner(self):
+        # check if user is owner/has permission to view all the poodle data
+        # if stealth voting is disabled this wont do anything
+        if not self.context.stealth_voting:
+            return True
+        if self.context.owner_info()['id'] == get_current().id:
```

Comment:
Please use plone.api.user.get_roles to check the owner-role (check the doc: http://docs.plone.org/develop/plone.api/docs/api/user.html for the api).